### DISCUSSION TITLE
禁用部分模组更新

### DISCUSSION THREAD

**Discussion by 18895kmxk** [Meloong-Git/PCL#4601] [讨论]
在模组列表中，选择全选和更新，会有一些模组旧版本比新版本跟好用，可以禁用该模组更新，获得更好的游玩体验

**Comment by wuliaodexiaoluo** [comment]
>会有一些模组旧版本比新版本跟好用

这个看个人喜好，你觉得旧版本好用，可能别人觉得新版本更好用

>可以禁用该模组更新

抱歉但是这不太可能实现，不能因为旧版本比新版本更好用就禁止更新，而且我们也明确表示 **不建议** 更新 Mod

**Comment by kong-que-dnf123** [comment]
~还有你分区投错了 这个不是自定义主页（~

**Comment by SALTWOOD** [comment]
分区错了，并且这个应该由个人体验决定。

**Comment by 18895kmxk** [comment]
由玩家确定不更新的模组，然后在该模组文件夹创建一个配置文件，玩家可自行配置自己不愿意更新的模组，读取配置后将不会搜取该模组更新信息

**Comment by WForst-Breeze** [comment]
#3901 是否可以满足期望呢

**Comment by 18895kmxk** [comment]
OK
